StringMethods.java

```diff
@@ -0,0 +1,287 @@
+/***
+ * Question: Create a ten string function.
+ * The available operations are listed below: Append, CountWords, Replace, isPalindrome, Splice, Split, MaxRepeatingCharacter, Sort, Shift, and Reverse.
+ * Owner name: Affan Sayeed.
+ * Date: 5-9-2024
+ */
+import java.util.Scanner;
+
+class StringFunction {
+    static String outputString = " ";
+

```

Comment:
add a comment explaining what this method does, what parameters it takes and what it returns